---
DebugHelperTest::ExceptionSub (message='test_exception_sub'):
  DebugHelperTest::ExceptionSub#message: Boo!
  DebugHelperTest::ExceptionSub#cause: 
